packages/serialization/json/src/jsonSerializationWriter.ts

```diff
@@ -43,41 +43,18 @@ export class JsonSerializationWriter implements SerializationWriter {
       isValuePresent &&
       this.writer.push(JsonSerializationWriter.propertySeparator);
   };
-  public writeNumberValue = (key?: string, value?: number): void => {
-    key && value && this.writePropertyName(key);
-    value && this.writer.push(`${value}`);
-    key && value && this.writer.push(JsonSerializationWriter.propertySeparator);
-  };
+  public writeNumberValue = (key?: string, value?: number): void => this.writeStringValue(key, value?.toString());
   public writeGuidValue = (key?: string, value?: Guid): void => {
     key && value && this.writePropertyName(key);
     value && this.writer.push(`"${value}"`);
     key && value && this.writer.push(JsonSerializationWriter.propertySeparator);
   };
-  public writeDateValue = (key?: string, value?: Date): void => {
-    key && value && this.writePropertyName(key);
-    value && this.writer.push(`"${value.toISOString()}"`);
-    key && value && this.writer.push(JsonSerializationWriter.propertySeparator);
-  };
-  public writeDateOnlyValue = (key?: string, value?: DateOnly): void => {
-    key && value && this.writePropertyName(key);
-    value && this.writer.push(`"${value.toString()}"`);
-    key && value && this.writer.push(JsonSerializationWriter.propertySeparator);
-  };
-  public writeTimeOnlyValue = (key?: string, value?: TimeOnly): void => {
-    key && value && this.writePropertyName(key);
-    value && this.writer.push(`"${value.toString()}"`);
-    key && value && this.writer.push(JsonSerializationWriter.propertySeparator);
-  };
-  public writeDurationValue = (key?: string, value?: Duration): void => {
-    key && value && this.writePropertyName(key);
-    value && this.writer.push(`"${value.toString()}"`);
-    key && value && this.writer.push(JsonSerializationWriter.propertySeparator);
-  };
-  public writeNullValue = (key?: string): void => {
-    key && this.writePropertyName(key);
-    this.writer.push(`null`);
-    key && this.writer.push(JsonSerializationWriter.propertySeparator);
-  };
+  public writeDateValue = (key?: string, value?: Date): void => this.writeStringValue(key, value?.toISOString());
+  public writeDateOnlyValue = (key?: string, value?: DateOnly): void => this.writeStringValue(key, value?.toString());
+  public writeTimeOnlyValue = (key?: string, value?: TimeOnly): void => this.writeStringValue(key, value?.toString());
+  public writeDurationValue = (key?: string, value?: Duration): void => this.writeStringValue(key, value?.toString());
+  public writeNullValue = (key?: string): void => this.writeStringValue(key, `null`);
```

Comment:
I believe this one shouldn't have been changed since null doesn't need quotes in JSON. Other methods are fine.